_stbt/wait.py

```diff
@@ -1,13 +1,20 @@
-"""Copyright 2015-2019 Stb-tester.com Ltd."""
+"""Copyright © 2015-2022 Stb-tester.com Ltd."""
 
 import functools
 import inspect
+from typing import Callable, TypeVar
 
 from .logging import debug
 
 
-def wait_until(callable_, timeout_secs=10, interval_secs=0, predicate=None,
-               stable_secs=0):
+T = TypeVar("T")
+
+
+def wait_until(callable_: Callable[[], T],
+               timeout_secs: int = 10,
+               interval_secs: int = 0,
```

Comment:
These should be `float`

```suggestion
               timeout_secs: float = 10,
               interval_secs: float = 0,
```

Reply:
Done.